### ISSUE TITLE
Error when trying to select a new weekly program for a zone

### ISSUE THREAD

**Issue by mezzarobba** [trahay/remora_programmer#2]
Hello,

Another error that I don't know enough to understand by myself. It seems to happen randomly (posssily when one makes too many requests in too little time, but I really don't know). 

```
nodejs[270]: 2018-11-22 07:39:12        saved program_semaine Bureau to database
nodejs[270]: 2018-11-22 07:39:19        { id: 4, name: 'Bureau', url: 'http://remora/fp4', program: 4 }
nodejs[270]: /home/domopi/opt/remora_programmer/node_modules/sqlite3/lib/trace.js:27
nodejs[270]:                     throw err;
nodejs[270]:                     ^
nodejs[270]: ReferenceError: error is not defined
nodejs[270]:     at Statement.db.run (/home/domopi/opt/remora_programmer/server.js:119:15)
nodejs[270]: --> in Database#run('update zones set name=?, url=?, program=? where id=?', 'Bureau', 'http://remora/fp4', '8', '4', [Function])
nodejs[270]:     at app.post (/home/domopi/opt/remora_programmer/server.js:117:5)
nodejs[270]:     at Layer.handle [as handle_request] (/home/domopi/opt/remora_programmer/node_modules/express/lib/router/layer.js:95:5)
nodejs[270]:     at next (/home/domopi/opt/remora_programmer/node_modules/express/lib/router/route.js:137:13)
nodejs[270]:     at Route.dispatch (/home/domopi/opt/remora_programmer/node_modules/express/lib/router/route.js:112:3)
nodejs[270]:     at Layer.handle [as handle_request] (/home/domopi/opt/remora_programmer/node_modules/express/lib/router/layer.js:95:5)
nodejs[270]:     at /home/domopi/opt/remora_programmer/node_modules/express/lib/router/index.js:281:22
nodejs[270]:     at Function.process_params (/home/domopi/opt/remora_programmer/node_modules/express/lib/router/index.js:335:12)
nodejs[270]:     at next (/home/domopi/opt/remora_programmer/node_modules/express/lib/router/index.js:275:10)
nodejs[270]:     at serveStatic (/home/domopi/opt/remora_programmer/node_modules/express/node_modules/serve-static/index.js:75:16)
```

**Comment by trahay** [comment]
Hi,
The error message " ReferenceError: error is not defined" comes from a typo in the code. I fixed it in commit  0b4a927. This will not fix the database problem, but at least the error message should help understanding what's wrong :)

**Comment by mezzarobba** [comment]
Thanks!
